hail/python/hailtop/batch/docker.py

```diff
@@ -78,15 +82,15 @@ def build_python_image(fullname: str,
     python3 -m pip check
 ''')
 
-            sync_check_shell_output(f'docker build -t {fullname} {docker_path}')
+            sync_check_exec('docker', 'build', '-t', fullname, docker_path, capture_output=(not show_docker_output))
```

Comment:
Why is it `(not show_docker_output)`? I would have thought it should be the opposite.

Reply:
`capture_output` means to collect the stdout/stderr with a pipe instead of inheriting the standard streams of the calling process. If we want to show the docker output on the user's stdout/err, we want not to capture the output.